v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+{{site.data.alerts.end}}
+
+## Conformance reporting tables
+
+The following new and updated tables are available for verifying constraint conformance.
+
+- [`system.replication_stats`](#system-replication_stats) shows information about whether data is under-replicated, over-replicated, or unavailable.
+- [`system.replication_constraint_stats`](#system-replication_constraint_stats) shows a list of violations to any data placement requirements you've configured, along with information about whether the cluster is actively recovering from those violations.
+- [`system.replication_critical_localities`](#system-replication_critical_localities) shows which localities in your cluster have data that is at risk in the event of node failures.
+- [`system.reports_meta`](#system-reports_meta) lists the IDs and times when the reports ran that generated the data in the `system.replication_*` tables.
+- [`crdb_internal.zones`](#crdb_internal-zones) can be used with the tables above to figure out the database and table names where the non-conforming or at-risk data is located.
+
+To configure how often the conformance reports run, adjust the XXX cluster setting.
+
+Only members of the `admin` role can access these tables. By default, the `root` user belongs to the `admin` role. For more information about users and roles, see [Manage Users](authorization.html#create-and-manage-users) and [Manage Roles](roles.html).
+
+{{site.data.alerts.callout_info}}
+The `system.replication_*` tables listed above are only populated in clusters where the zone configurations have been changed from the defaults.
+{{site.data.alerts.end}}
+
+### system.replication_stats
+
+The `system.replication_stats` table shows information about whether data is under-replicated, over-replicated, or unavailable.
+
+For an example using this table, see [Find out which databases and tables have under-replicated ranges](#find-out-which-databases-and-tables-have-under-replicated-ranges).
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM system.replication_stats;
+~~~
+
+| Column name             | Data type          | Description                                                                            |
+|-------------------------+--------------------+----------------------------------------------------------------------------------------|
+| zone_id                 | [`INT8`](int.html) | The ID of the [replication zone](configure-zone.html).                                 |
+| subzone_id              | [`INT8`](int.html) | The ID of the subzone (i.e., [partition](partition-by.html)).                          |
+| report_id               | [`INT8`](int.html) | The ID of the [report](#system-reports_meta) that generated this data.                 |
+| total_ranges            | [`INT8`](int.html) | Total [ranges](architecture/overview.html#architecture-range) in the cluster.          |
```

Comment:
Not in the cluster! In the zone that this report entry refers to (and all the child ranges that don't get their own report entry cause they haven't overwritten the respective attributes).